README.md

```diff
@@ -135,6 +136,7 @@ See also https://github.com/timotheecour/D_vs_nim/issues/11
 | increment | i++ | i+=1 or inc(i) |
 | concatenation | ~ | & |
 | empty statement | {} | discard |
+| null pointer | `null` | `nil` |
 | **syntax:parsing** |
 | alias | `alias T2=T;` | ?; template, see https://github.com/nim-lang/Nim/issues/7090 |
```

Comment:
> experimental not nil type annotation.

right, i'm not aware of CT enforced type annotation like not nil in D

> I plan to make a PR for Nim to move the not nil documentation to the correct section of the manual.

indeed, shd be in experimental feature section

Reply:
I made the PR to move Nim’s `not nil` documentation to the Experimental Features section of the manual: https://github.com/nim-lang/Nim/pull/14027